Unhandled error in ``create_ini()``

This request:
http://tools.pacificclimate.org/dataportal/data/pcds/agg/?from-date=1870%2F01%2F01&to-date=2017%2F07%2F31&input-polygon=&input-var=&network-name=FRBC&input-freq=&data-format=ascii&cliptodate=cliptodate&download-timeseries=Timeseries

leads to this unhandled exception:

```
[2017-07-26 11:18:12 +0000] [27692] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 79, in ziperator
    for name, responder in responders:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pdp_util/agg.py", line 166, in get_pcds_responders
    yield (name, handler(newenv, null_start_response))
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pydap/handlers/pcic/__init__.py", line 70, in __call__
    s = self.create_ini(sesh, net_name, native_id)
  File "/home/websites/tools.pacificclimate.org/data_portal-2.4.2/env/local/lib/python2.7/site-packages/pydap/handlers/pcic/__init__.py", line 144, in create_ini
    ''' % locals()
TypeError: a float is required
```

Is it possible that stations without location (either x/y or elevation) metadata are being selected and not guarded against?

--- a/pydap/handlers/pcic/__init__.py
+++ b/pydap/handlers/pcic/__init__.py
@@ -105,6 +105,7 @@ class PcicSqlHandler(object):
                 q = q.filter(History.sdate == sdate)
 
             _, station_name, network, geom, elevation = q.first()
+            elevation = elevation if elevation else float('nan')
             lat, lon = (sesh.scalar(geom.y), sesh.scalar(geom.x)) if geom else (float('nan'), float('nan'))
 
         dsn = self.dsn

--- a/setup.py
+++ b/setup.py
@@ -11,7 +11,7 @@ __version__ = (0, 0, 8)
 class PyTest(TestCommand):
     def finalize_options(self):
         TestCommand.finalize_options(self)
-        self.test_args = '-v --tb=no -m "not poor_unittest" tests'
+        self.test_args = ["-v", "--tb=no", "-m", "not poor_unittest", "tests"]
         self.test_suite = True
 
     def run_tests(self):

--- a/tox.ini
+++ b/tox.ini
@@ -2,7 +2,7 @@
 envlist = py27
 sitepackages = False
 indexserver = 
-    default = http://tools.pacificclimate.org/pypiserver/
+    default = https://pypi.pacificclimate.org/simple/
 
 [testenv]
 deps = -rtest_requirements.txt

